src/agent/srcview/src/report.rs

```diff
@@ -347,7 +355,7 @@ impl Report {
 
         let unixtime = SystemTime::now()
             .duration_since(UNIX_EPOCH)
-            .expect("system time before unix epoch, wtf")
```

Comment:
:(

Reply:
panic bad?

Also, we have to pass the `GeoPol` tool's analysis.   For reference: https://aka.ms/NExTGeoPol